students/eric_grandeo/lesson01/test_unit.py

```diff
@@ -0,0 +1,130 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+from unittest.mock import patch
+import io
+import sys
+sys.path.append('./inventory_management')
```

Comment:
Since you're adding _inventory_management_ to your unit and integration tests, you don't need to do the relative imports you have in other files.